src/ages/ages.types.ts

```diff
@@ -4,6 +4,7 @@ import { MarketEmission } from "../utils";
 import { AgeDistribution } from "./distributions/distributions.types";
 
 export interface EpochConfig {
+  id: string;
```

Comment:
I thought the `epochNumber` was used as an id

Reply:
I removed `epochNumber`, it's not needed in the end, and the code didn't compile

Reply:
The ID is `age4` or `age2-epoch2` for example.